arcgis-ios-sdk-samples/Maps/Create and save a map/SaveAsViewController.swift

```diff
@@ -38,12 +49,81 @@ class SaveAsViewController: UITableViewController {
             presentAlert(message: "Please enter a title.")
             return
         }
-        
+        // Get the tags from the text field.
         let tags = tagsTextField.text?
             .components(separatedBy: ",")
             .map { $0.trimmingCharacters(in: .whitespacesAndNewlines) } ?? []
-        
+        // Get the item description from the text field.
         let itemDescription = descriptionTextField.text ?? ""
-        delegate?.saveAsViewController(self, didInitiateSaveWithTitle: title, tags: tags, itemDescription: itemDescription)
+        
+        delegate?.saveAsViewController(self, didInitiateSaveWithTitle: title, tags: tags, itemDescription: itemDescription, folder: selectedFolder)
+    }
+    
+    /// Toggles visisbility of the reference scale picker.
+    func toggleFolderPickerVisibility() {
+        tableView.performBatchUpdates({
+            if folderPickerHidden {
+                folderLabel.textColor = .accentColor
+                tableView.insertRows(at: [folderPickerIndexPath], with: .fade)
+                folderPickerHidden = false
```

Comment:
FYI: `folderPickerHidden.toggle()` could have been used after the `if` statement instead.